components/Snackbar/src/MDCSnackbarMessage.h

```diff
@@ -146,6 +146,14 @@ extern NSString *__nonnull const MDCSnackbarMessageBoldAttributeName;
  */
 @property(nonatomic, readonly, nullable) NSString *voiceNotificationText;
 
+/*
+ Determines if an @c MDCSnackbar buttons should use the @c MDCRippleView behavior or not.
+ By setting this property to @c YES, @c MDCRippleView is used to provide the user visual
+ touch feedback, instead of the legacy @c MDCInkView.
+ @note Defaults to @c NO.
```

Comment:
For this API and all of the others it may be helpful for clients to indicate that this API will eventually be enabled, deprecated, and then deleted as part of our general migration. I also recommend not including class names in these APIs because they reveal implementation details that may change over time. E.g.:

```
By setting this property to @c YES, the Ripple component will be used instead of Ink
to display visual feedback to the user.

@note This property will eventually be enabled by default, deprecated, and then deleted as part
of our migration to Ripple.

Defaults to NO.
```

Also note that "Defaults" statements aren't typically shown as notes.

Reply:
Great update to the copy, thanks!